fastlane/Fastfile

```diff
@@ -77,4 +79,23 @@ private_lane :ensure_assets_location do
     UI.error("File '#{path}' shouldn't be in this directory, only `.md` files are allowed")
   end
   UI.user_error!("Found some files in a wrong directory") if error
+  UI.success "✅  All assets are in the correct subfolder"
+end
+
+private_lane :ensure_fastlane_spelling do
```

Comment:
Small complaint would be that this is about all tools, not just fastlane, and it's really more about formatting rather than spelling .

`ensure_tool_name_formatting` ?